tests/cartservice/cartservice.tests.csproj

```diff
@@ -1,17 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Protobuf" Version="3.6.0" />
-    <PackageReference Include="Google.Protobuf.Tools" Version="3.6.0" />
-    <PackageReference Include="Grpc" Version="1.12.0" />
-    <PackageReference Include="Grpc.Tools" Version="1.12.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
```

Comment:
Thanks for caching this. Should we add a test command to the Dockerfile to catch tests/* issues? (feel free to open a backlog item)

Reply:
I prefer to go in more traditional route: adding tests that need to pass on every checking, for the PR to be merged.

Reply:
Docker build runs on every PR. That's why I was suggesting to put it in docker build. It's not a rare practice to do that. Also since our dockerfiles are multi-stage, it wouldn't have a bad impact.

Reply:
Otherwise it's really really really complicated (if possible at all) to configure TravisCI for all languages we use. TravisCI config assumes 1 lang per repo: https://docs.travis-ci.com/user/languages/

Reply:
Sorry, was busy with something else.
To your original question, As we are directly depending on the project, these references are redundant.

Reply:
@simonz130 can you confirm? if that's the case I'll merge this.
ideally we should try to move tests for this service to `src/cartservice` so we can run it as part of `docker build` process.